assets/js/form.js

```diff
@@ -0,0 +1,21 @@
+const contactForm = document.getElementById('form');
+const email = document.getElementById('contact-email');
+const errorMessage = document.getElementById('email-message');
+
+contactForm.addEventListener('submit', (event) => {
+  event.preventDefault();
+
+  const emailRegExp = /^[a-z_\-0-9.*#$!~%^&-+?|]+@+[a-z\-0-9]+(.com)$/gm;
+  const emailIsValid = emailRegExp.test(email.value);
+
+  if (!emailIsValid) {
+    errorMessage.style.display = 'block';
+    email.style.borderBottom = '1px solid #e78587';
+    errorMessage.innerHTML = 'Invalid Email!<br>Please enter a valid email in LowerCase';
+  } else {
```

Comment:
- [ ] Your form validation is working correctly, but according to project requirements, the error message should show up close to the "submit" button. Please make this change 👍 
![image](https://user-images.githubusercontent.com/46839548/168162127-c2d94be7-5a56-4ffe-bf40-fc0c27760ebe.png)